workflow-service/src/main/java/com/redhat/parodos/workflow/execution/dto/WorkFlowContextResponseDTO.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2022 Red Hat Developer
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.redhat.parodos.workflow.execution.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/**
+ * Workflow context parameters response dto
+ */
+
+@Builder
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class WorkFlowContextResponseDTO {
+
+	private String workFlowExecutionId;
+
+	@JsonInclude(JsonInclude.Include.NON_EMPTY)
+	private WorkFlowOptionsResponseDTO workFlowOptions;
```

Comment:
it looks like this only supports for assessment options,  should we extend it to all publicly visiable fields in workContext?

Reply:
Let's do it gradually as needed by the client, else, we'll have to commit to an API that might not be needed, but we'll have to maintain.
We can start with option only, since extending this API doesn't consider breaking it and it will maintain its backward compatibility.

Reply:
ok, let's start with options